src/test/DAL/MetaQuestion.test.js

```diff
@@ -254,6 +260,51 @@ describe('MetaQuestionRepository happy path tests', () => {
 
         expect(addedAnswers.length).toBe(0);
     });
+
+    test('delete appendix, make sure MQs still exist with null appendixTag', async () => {
```

Comment:
I may have missed it,
But we dont work to work with null, because we had an issues with null before. please make it undefined

Reply:
This is db behavior in case the MQ isn't attached to any appendix. 
Since this is db behavior I have to test it that way

Reply:
I added a test for that. anyway if `null` values cause problems at FE we can fix the returned object at `mqBusinessToFE` in the facade

Reply:
Fine by me, but i think it should be done in this PR

Reply:
I'll do it if such a bug comes up. It's quick work.

Reply:
fixed